pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -149,17 +149,30 @@ func (config *V1RegistriesConf) Nonempty() bool {
 		len(config.V1TOMLConfig.Block.Registries) != 0)
 }
 
+// aliasesConf includes a map of aliases.
```

Comment:
Maybe
```suggestion
// aliasesConf is the subset of the V2RegistriesConf format that is software-maintained in aliases.conf
```
or something vaguely like that?